yeonjae/programmers-1/음양더하기.js

```diff
@@ -0,0 +1,15 @@
+function solution(absolutes, signs) {
+  let result = 0;
+
+  for (let i = 0; i < absolutes.length; i++) {
+    signs[i] ? (result += absolutes[i]) : (result -= absolutes[i]);
+  }
+
+  return result;
+}
```

Comment:
```suggestion
function solution(absolutes, signs) {
  let result = 0;

  for (let i = 0; i < absolutes.length; i++) {
    result += signs[i] ? absolutes[i] : -absolutes[i];
  }

  return result;
}
```
- 잘푸셨네요 고생하셨습니다 👍 삼항 연산자 내에서 result += absolutes[i] 이렇게 해주기보다 위에 예제처럼 해주시는게 좀 더 깔끔해보이고 동일한 결과가 나옵니다 ! 
- 아래와 같은 reduce를 활용한 풀이도 가능합니다 :) 참고해주세요~
```js
function solution(absolutes, signs) {
  return absolutes.reduce((acc, cur, idx) => {
    const sign = signs[idx] ? 1 : -1;
    return (acc += cur * sign);
  }, 0);
}
```

Reply:
확인했습니다 !!